<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-01-13T17:54:55 IST" name="Default suite" finished-at="2024-01-13T17:55:03 IST" duration-ms="7756">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:cucumber.TestNGTestRunner@17f9d882]" name="runScenario" class="io.cucumber.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test started-at="2024-01-13T17:54:55 IST" name="Default test" finished-at="2024-01-13T17:55:03 IST" duration-ms="7756">
      <class name="cucumber.TestNGTestRunner">
        <test-method is-config="true" signature="setUpClass(org.testng.ITestContext)[pri:0, instance:cucumber.TestNGTestRunner@17f9d882]" started-at="2024-01-13T17:54:55 IST" name="setUpClass" finished-at="2024-01-13T17:54:57 IST" duration-ms="1118" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@254f906e]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:cucumber.TestNGTestRunner@17f9d882]" started-at="2024-01-13T17:54:57 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2024-01-13T17:55:03 IST" duration-ms="6416" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["Title of your scenario outline"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Error validation"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Incorrect email or password.] but found [Incorrect email password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Incorrect email or password.] but found [Incorrect email password.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at rahulshettyacademy.stepDefinitions.StepDefinitionImpl.something_message_is_displayed(StepDefinitionImpl.java:64)
at âœ½."Incorrect email password." message is displayed(file:///C:/Users/Hp/eclipse-workspace/SeleniumTraining/SeleniumFrameworkDesign/src/test/java/cucumber/ErrorValidations.feature:12)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method is-config="true" signature="tearDownClass()[pri:0, instance:cucumber.TestNGTestRunner@17f9d882]" started-at="2024-01-13T17:55:03 IST" name="tearDownClass" finished-at="2024-01-13T17:55:03 IST" duration-ms="115" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- cucumber.TestNGTestRunner -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
